_posts/2021-01-25-final-report-fall2020-kayala.md

```diff
@@ -0,0 +1,42 @@
+# Final Report - Fall 2020 Semester #
+
+## Activities/Accomplishments and Concepts/Lessons Learned ##
+Concepts:
+- Working with Unreal Engine
+    - LOD and Materials
+    - Movement and Cameras
+    - Blueprints
+- Creating Levels
+    - Perlin noise generated maps
+    - Transfer to Unreal Engine 4.25
+
+Activities:
+I started the semester just by learning about Unreal Engine. I spent time working with making new materials and 
+objects, learning about Level Of Detail(LOD) mechanics and best practices for making hyper realistic environments. Part of that was learning about how to efficiently show realistic environments, and that hinged around only loading what is necessary.
+
+
+![LOD 1](../assets\2021-01-25-final-report-fall2020-kayala\lod1Rock.png)
+![LOD 5](../assets\2021-01-25-final-report-fall2020-kayala/lod5rock.png)
+
+In the picture above, see the difference between a LOD of 1, which renders when the camera is close to the object, versus a LOD 5 object, which is used when the camera is further. 
+
+Then, I moved on to moving actors, including humanoid and rover forms. Moving the cameras and having them positioned and scripted correcly for 1st person views that were necessary to trianing the model was especially difficult. Here's what it looked like on the car model:
+![](../assets\2021-01-25-final-report-fall2020-kayala/vehicle-cam.png)
+
+Finally, I moved onto working with Blueprints, since a large part of the work involved taking and organizing screenshots at appropriate points in the movement of the vehicle. I struggled with this because it was a very large environment with not-so-adequate documentation, but eventually was able to learn how to take these screenshots and later organize them with a script into a csv. 
```

Comment:
```suggestion
Finally, I moved onto working with Blueprints, since a large part of the work involved taking and organizing screenshots at appropriate points in the movement of the vehicle. I struggled with this because UE is a very large program with not-so-adequate documentation for some tasks, but I was eventually able to take these screenshots and later organize them with a script into a CSV file. 
```